content/kubeone/v1.7/architecture/operating-system-manager/_index.en.md

```diff
@@ -52,4 +52,4 @@ For each MachineDeployment we have two types of configurations, which are stored
 1. **Bootstrap**: Configuration used for initially setting up the machine and fetching the provisioning configuration.
 2. **Provisioning**: Configuration with the actual `cloud-config` that is used to provision the worker machine.
 
-![Architecture](/img/kubeone/v1.7/operating-system-manager/architecture.png?classes=shadow,border "Architecture")
+![Architecture](/img/kubeone/main/operating-system-manager/architecture.png?classes=shadow,border "Architecture")
```

Comment:
Can we drop the `main` from the URL (i.e. move the image somewhere outside that directory) because we will mostly like replace it again the next time we create docs for a new minor release?

Reply:
I did that with this PR now, because I agree that having the images in content/ is much, much easier.